tests/providers/google/cloud/transfers/test_gdrive_to_gcs_system.py

```diff
@@ -1,40 +0,0 @@
-#
```

Comment:
Why this file was removed, since you migrated example_life_sciences?

Reply:
I restored the file